src/Core/Compiler/CompilerService.cs

```diff
@@ -454,7 +466,7 @@ public IEnumerable<QsNamespaceElement> IdentifyElements(string source, ITaskRepo
                     var tempPath = Path.GetTempFileName();
                     var bcFile = CompilationLoader.GeneratedFile(tempPath, Path.GetDirectoryName(tempPath), ".bc", "");
                     var diagnostics = new List<IRewriteStep.Diagnostic>();
-                    CompilationSteps.GenerateBitcode(qsCompilation, capability, bcFile, diagnostics: diagnostics);
+                    CompilationSteps.GenerateBitcode(qsCompilation, assebmlyConstants, bcFile, diagnostics: diagnostics);
```

Comment:
Is this misspelled?

Reply:
fixed, and found that there is already an assemblyConstatns from the UpdateCompilation method. I moved the changes to the assemblyConstants to that function to keep it all in one place.